packages/host/src/lib/serviceDiscovery/sd-adapter.ts

```diff
@@ -94,13 +94,24 @@ export class ServiceDiscovery {
                 throw new Error("Content-type mismatch");
             }
             this.logger.debug("Topic routed:", config);
+
             return topic;
         }
 
         this.logger.debug("Topic added:", config);
         const origin: StreamOrigin = { id: this.hostName, type: "hub" };
         const newTopic = new Topic(topicName, config.contentType, origin);
 
+        if (cli) {
```

Comment:
send it always, sd discover can't know about CLI

Reply:
createTopicIfNotExist is already mentioned in two other methods which already sends the info to cpm, am I supposed to move these to the createTopicIfNotExist ?

Reply:
do not mix responsibility - createTopicIfNotExist creates topic. 
Create a method to send topicInfo or call sendTopicInfo directly after topic is created - after createTopicIfNotExist is called